examples/index.js

```diff
@@ -93,6 +93,21 @@ async function switchChain() {
   }
 }
 
+async function addToken() {
```

Comment:
```
ethereum.request({
  method: 'wallet_watchAsset',
  params: {
    type: 'ERC20',
    options: {
      address: '0xb60e8dd61c5d32be8058bb8eb970870f07233155',
      symbol: 'FOO',
      decimals: 18,
      image: 'https://foo.io/token-image.svg',
    },
  },
});
```
It's like this according to EIP.

Reply:
The address will be in options key, and params will be an object.

Reply:
https://github.com/arcana-network/auth/pull/127/commits/3b7d875e916076f704692a66cf1ec3794762f476